docs/generate_modules.md

```diff
@@ -1,6 +1,6 @@
 # Generate Modules
 
-This document shows the procedure for generating modules by yourself.  
```

Comment:
Please do not delete last spaces, too.
The last spaces is needed to add return in html rendering.
Also there are same cases in this file.

Reply:
reverted